subsys/net/ip/Kconfig

```diff
@@ -304,6 +304,15 @@ config NET_UDP_CHECKSUM
 	  Enables UDP handler to check UDP checksum. If the checksum is invalid,
 	  then the packet is discarded.
 
+config NET_UDP_MISSING_CHECKSUM
```

Comment:
I'd say the naming of this option is a bit unclear. Something like NET_UDP_REQUIRE_IPV4_CHECKSUM would be better IMHO. (That's assuming we need such an option at all, why btw?)

Reply:
Your proposal seems even more cryptic.

Reply:
My proposal says "REQUIRE". Whereas you have "UDP_MISSING_CHECKSUM". Ok, missing checksum, what about missing checksum? That said, it's all just suggestions to draw attention to, up to you to see if anything needs changing.

Reply:
I think @tobur suggestion is better as then we can leave the default 'n', with **require** the default would be 'y' in which case the setting needs to be always in .config file.